app/controllers/PdfDownloadController.scala

```diff
@@ -42,10 +42,20 @@ class PdfDownloadController @Inject()(appConfig: FrontendAppConfig,
         pdfService.generatePdf(applicationPdf(appConfig, c, attachmentData)).map { binaryFile =>
           Results.Ok(binaryFile.content)
             .as(binaryFile.contentType)
-            .withHeaders(CONTENT_DISPOSITION -> s"filename=confirmation_$reference.pdf")
+            .withHeaders(CONTENT_DISPOSITION -> s"filename=BTIConfirmation$reference.pdf")
```

Comment:
I don't which are the standards / conventions  for file name. Is this name ok ? BTIConfirmation$reference.pdf

Reply:
I checked on the file names with Matt yesterday.   He preferred / was happy with the new format